tests/test_features/test_base.py

```diff
@@ -47,3 +48,57 @@ def test_subclass(self):
     #     visualizer = FeatureVisualizer()
     #     with self.assertRaises(NotImplementedError):
     #         visualizer.poof()
+    
+class DataVisualizerBaseTests(MultiFeatureVisualizer):
+    
+    def test_determine_target_color_type(self):
```

Comment:
I realize that the manifold tests had this as one long test; but now that we've broken it out into independent functionality, perhaps we could turn this into more tests:

1. Test single (both auto and specified)
2. Test discrete (both auto and specified)
3. Test continuous (both auto and specified) 
4. Test bad target types raise an exception 

That would allow us to test other things as well such as `classes_`, `range_` and `_colors` and be a bit more thorough with the testing process (e.g. use `make_classification` and `make_regression` instead of `np.random`).

Reply:
WIll do it.